src/PKSim.Core/Services/ISimulationEngine.cs

```diff
@@ -14,18 +14,7 @@ public interface ISimulationEngine<TSimulation> : ISimulationEngine where TSimul
       ///    Run the simulation asynchronously (Hand returns right away to the caller)
       /// </summary>
       /// <param name="simulation">simulation to run</param>
-      Task RunAsync(TSimulation simulation);
-
-      /// <summary>
-      ///    Run the simulaton synchronously and does not raise any UI events
-      /// </summary>
-      /// <param name="simulation">Simulation to run</param>
-      void Run(TSimulation simulation);
```

Comment:
Not required anymore

Reply:
And this was only used in test. Now tests are also using the same RunAsync method. All good